### PR TITLE
add job definition for hyp3-insar-gamma

### PR DESCRIPTION
**Pull request by Jlrine2**
Must add secret for insar gamma job definition and create pull permissions for hyp3-v2 account in order for job definition to work

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -77,6 +77,7 @@ jobs:
                 EDLUsername="${EDL_USERNAME}" \
                 EDLPassword="${EDL_PASSWORD}" \
                 RtcGammaImage="${RTC_GAMMA_REPOSITORY}:${IMAGE_TAG}" \
+                InsarGammaImage="${INSAR_GAMMA_REPOSITORY:${IMAGE_TAG" \
```
Comment (asjohnston-asf) [inline .github/workflows/deploy.yml:80]: closing `}`

Comment (asjohnston-asf) [reply .github/workflows/deploy.yml:80]: twice

Comment (asjohnston-asf) [reply .github/workflows/deploy.yml:80]: add `INSAR_GAMMA_REPOSITORY: 626226570674.dkr.ecr.us-east-1.amazonaws.com/hyp3-insar-gamma`

**Comment by asjohnston-asf** [comment]
I've added permissions for the hyp3-v2 account to pull from the hyp3-insar-gamma ecr repo

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -10,6 +10,7 @@ env:
   AWS_REGION: us-west-2
   TEMPLATE_BUCKET: cf-templates-aubvn3i9olmk-us-west-2
   RTC_GAMMA_REPOSITORY: 626226570674.dkr.ecr.us-east-1.amazonaws.com/hyp3-rtc-gamma
+  INSAR_GAMMA_REPOSITORY: 626226570674.dkr.ecr.us-east-1.amazonaws.com/hyp3-insar-gamma
```
[the remark this answers is not in the document]
Comment (Jlrine2) [reply .github/workflows/deploy.yml:13]: I like it

**Pull request merged**